src/com/lftechnology/java/training/krishna/basics/even_fibonacci/EvenFibonacciDemo.java

```diff
@@ -0,0 +1,15 @@
+package com.lftechnology.java.training.krishna.basics.even_fibonacci;
+
+import java.util.logging.Logger;
+
+
+public class EvenFibonacciDemo {
+	private final static Logger LOGGER = Logger.getLogger(EvenFibonacci.class.getName());
+	public static void main(String[] args) {
+		EvenFibonacci evenFibonacci = new EvenFibonacci(0, 1);
```

Comment:
I don't think passing 0,1 as constructor make any sense here. What happens when you pass 11,12 instead of the 0,1? You should either validate the given numbers falls under Fibonacci series or else drop this idea of passing predefined variable here. 

I would have loved if you have implemented the validity of inputted numbers as part of Fibonacci Series  in the Constructor. Or, may be you could let user to set the MAX_LIMIT instead?